client/ionic/package.json

```diff
@@ -31,11 +31,13 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test",
+    "test": "npm run lint && react-scripts test",
     "eject": "react-scripts eject",
     "check": "gts check",
     "cleanbuild": "gts clean",
     "compile": "tsc -p .",
+    "typecheck": "tsc --noEmit -p .",
```

Comment:
Do we need this given that `gts check` (I believe) already does a typecheck?

Reply:
Right, actually from what I can tell it's `pretest` that runs `compile` (the line above) which catches the type errors during CI (although I'm not sure if `gts check` also runs this). I added this just to expose a way to run the typecheck at the cli, but not sure if it's necessary.